From 424fc1e6244f28690c9ea0b99f06158273732a28 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 13 Nov 2016 16:21:42 +0200
Subject: [PATCH 27/58] ARM: i.MX6: update defconfig

1) Enable SCSI GENERIC support.
2) Enable GADGET UAC1.
3) Disable ATH6KL module support.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/configs/cm_fx6_defconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/cm_fx6_defconfig b/arch/arm/configs/cm_fx6_defconfig
index db3a9e0..407319a 100644
--- a/arch/arm/configs/cm_fx6_defconfig
+++ b/arch/arm/configs/cm_fx6_defconfig
@@ -173,6 +173,7 @@ CONFIG_EEPROM_AT24=y
 CONFIG_EEPROM_AT25=y
 # CONFIG_SCSI_PROC_FS is not set
 CONFIG_BLK_DEV_SD=y
+CONFIG_CHR_DEV_SG=m
 CONFIG_SCSI_CONSTANTS=y
 CONFIG_SCSI_LOGGING=y
 CONFIG_SCSI_SCAN_ASYNC=y
@@ -199,8 +200,6 @@ CONFIG_SMC911X=y
 CONFIG_SMSC911X=y
 # CONFIG_NET_VENDOR_STMICRO is not set
 CONFIG_ATH_CARDS=y
-CONFIG_ATH6KL=m
-CONFIG_ATH6KL_SDIO=m
 CONFIG_MWIFIEX=m
 CONFIG_MWIFIEX_SDIO=m
 # CONFIG_INPUT_MOUSEDEV_PSAUX is not set
@@ -331,6 +330,7 @@ CONFIG_USB_GADGET=y
 CONFIG_USB_FSL_USB2=y
 CONFIG_USB_ZERO=m
 CONFIG_USB_AUDIO=m
+CONFIG_GADGET_UAC1=y
 CONFIG_USB_ETH=m
 CONFIG_USB_MASS_STORAGE=m
 CONFIG_USB_G_SERIAL=m
-- 
1.9.1

